src/coprocessor/dag/executor/aggregation.rs

```diff
@@ -168,7 +158,7 @@ impl Executor for HashAggExecutor {
         }
 
         match self.group_key_aggrs.get_index_mut(self.cursor) {
-            Some((group_key, aggrs)) => {
+            Some((mut group_key, aggrs)) => {
                 self.cursor += 1;
                 let mut aggr_cols = Vec::with_capacity(2 * self.aggr_func.len());
```

Comment:
Why not alloc space for `group key` as `StreamAgg` does?

Reply:
Since the `group key` in `HashAgg` is a `Vec<u8>` while in `StreamAgg` is `Vec<Datum>`.